### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by imanishbarnwal**
Linked CODEOFCONDUCT.md to README.md
Issue: #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -31,7 +31,8 @@ Visit `[url of project]/messages` and enter the number of the whitelisted phone
 
 We love to hear from you so if you have questions, comments or find a bug in the project, let us know! You can either:
 
-* Open an issue on this repository
+* Open an issue on this repository. 
+* Before you start contributing, check out the [Code of Conduct](CODEOFCONDUCT.md).
```
Comment (lukeocodes) [inline README.md:35]: hey @imanishbarnwal this isn't actually a valid URL, can you change it and test so that you know it works?

**Review by lukeocodes — CHANGES_REQUESTED** [review]
thanks for your contribution, please make requested change

File: README.md (hunk 1 above)
Comment (imanishbarnwal) [reply README.md]: Oh so sorry, thanks for highlighting this I just updated this. Kindly check it now.

**Comment by imanishbarnwal** [comment]
Hello @lukeocodes, can you please review my recent update?

**Review by lukeocodes — APPROVED** [review]

**Pull request merged**